Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/MenuActionBuilder.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Activities.DesignViewModels;
+using System.Activities.ViewModels;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UiPath.Shared;
+
+namespace UiPath.Cryptography.Activities.NetCore
```

Comment:
I assume this is taken from the Activities repo. If so, please leave a note.

Reply:
Left a note